include/deal.II/fe/fe_values.h

```diff
@@ -292,6 +330,24 @@ namespace FEValuesViews
                               std::vector<typename ProductType<value_type,typename InputVector::value_type>::type> &values) const;
 
     /**
+     * Same as above, but using a of vector local degree-of-freedom values.
+     *
```

Comment:
Sure, will do. I believe that these functions supports both `std::vector` and `Arrayview`, which we use internally.